source/features/linkify-urls-in-code.tsx

```diff
@@ -14,7 +14,7 @@ export const linkifiedURLClass = 'rgh-linkified-code';
 const currentRepo = getOwnerAndRepo();
 const options = {
 	user: currentRepo.ownerName || '/',
-	repo: currentRepo.repoName || '/',
+	repository: currentRepo.repoName || '/',
```

Comment:
Just need to sort out the signature for the `fn` parameter as the options are different.

```bash
ERROR in /Users/nickytonline/dev/refined-github/source/features/linkify-urls-in-code.tsx
[tsl] ERROR in /Users/nickytonline/dev/refined-github/source/features/linkify-urls-in-code.tsx(35,21)
      TS2349: Cannot invoke an expression whose type lacks a call signature. Type '{ (string: string, options: TypeDomOptions): DocumentFragment; (string: string, options?: Options | undefined): string; } | { (string: string, options: TypeDomOptions): DocumentFragment; (string: string, options?: Options | undefined): string; }' has no compatible call signatures.
```